src/pages/collection/Series.tsx

```diff
@@ -63,6 +63,13 @@ const Series = () => {
 
   const dispatch = useDispatch();
 
+  const languageMapping = { 'x-jat': 'ja', 'x-kot': 'ko', 'x-zht': 'zh-hans' };
+  const mainTitle = series.AniDB?.Titles.find(title => title.Type === 'Main');
+  const mainTitleLanguage = mainTitle?.Language ?? '';
+  const originalTitle = series.AniDB?.Titles.find(title => title.Language === languageMapping[mainTitleLanguage]);
+  const mainTitleCheck = mainTitle?.Name !== series.Name ? <span>{mainTitle?.Name}</span> : null;
+  const originalTitleCheck = originalTitle?.Name !== series.Name ? <span>{originalTitle?.Name}</span> : null;
```

Comment:
This can be a single `useMemo` hook with dependency on `series`.
Everything except `mainTitleCheck` and `originalTitleCheck` shouldn't exist outside the hook.

Reply:
Rename the hook output variables to `mainTitle` and `originalTitle`

Reply:
Seem to work still 👍